steps/nvm/0.1.0/step.yml

```diff
@@ -0,0 +1,37 @@
+title: NVM
+summary: Install NVM and select Node version
+description: |
+  Install NVM and select Node version
+  NVM = Node Version Manager - Simple bash script to manage multiple active node.js versions
+  See https://github.com/creationix/nvm
+website: https://github.com/almouro/bitrise-nvm-step
+source_code_url: https://github.com/almouro/bitrise-nvm-step
+support_url: https://github.com/almouro/bitrise-nvm-step/issues
+published_at: 2016-07-02T22:15:30.551754569+02:00
+source:
+  git: https://github.com/Almouro/bitrise-nvm-step.git
+  commit: 39aa1b7d154d4498b16a5eb6a0f9d13bc479d62c
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
```

Comment:
I'd remove the `ios` project type filter, this step can work with other types of projects too